src/fragments/start/getting-started/reactnative/getting-started-steps-ui.mdx

```diff
@@ -4,7 +4,7 @@
 Install the necessary dependencies by running the following command:
 
 ```sh
-npm install aws-amplify amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage @azure/core-asynciterator-polyfill
+npm install aws-amplify@^5 amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage @azure/core-asynciterator-polyfill
```

Comment:
Is this fragment used in v5 or v6? The path doesn't seem specific to v5

Reply:
yeah this is v5 because we have a reference to `amazon-cognito-identity-js`